.env.structure.example

```diff
@@ -0,0 +1,13 @@
+# Copy this file to a new file named '.env' to automatically load these
+# values as environment variables when running the Structure.
+
+# The Agent example included expects an OpenAI key. Supply yours here:
+OPENAI_API_KEY=YOUR_KEY_HERE
+
+# The event driver expects a Griptape API Key as a parameter in order to emit events.
```

Comment:
`as a parameter in order to emit events` -> `as a parameter in order to emit events back to Griptape Cloud`

Reply:
think I got it